build.gradle.kts

```diff
@@ -59,6 +59,13 @@ tasks {
 repositories {
     mavenCentral()
     mavenLocal()
+    maven {
```

Comment:
Har endra i OpenHTMLtoPDF så versjonar blir publisert til GitHub. Bør nok etablere ein pipeline for å pushe til Maven central også, men det er litt meir greier som må på plass først, så vil ikkje vente på det.